cmake/sip_helper.cmake

```diff
@@ -92,6 +99,7 @@ function(build_sip_binding PROJECT_NAME SIP_FILE)
     add_custom_command(
         OUTPUT ${SIP_BUILD_DIR}/Makefile
         COMMAND ${PYTHON_EXECUTABLE} ${sip_SIP_CONFIGURE} ${SIP_BUILD_DIR} ${SIP_FILE} ${sip_LIBRARY_DIR} \"${INCLUDE_DIRS}\" \"${LIBRARIES}\" \"${LIBRARY_DIRS}\" \"${LDFLAGS_OTHER}\" \"${EXTRA_DEFINES}\"
+        COMMAND sed -i 's/ -I/ -isystem/g' ${SIP_BUILD_DIR}/Makefile
```

Comment:
I think there are windows users, and it seems like requiring `sed` would be a problem there. Can we do this in `sip_configure.py` [after generating the makefile](https://github.com/ros-visualization/python_qt_binding/blob/fd4d50b6e2b65e329039fba033cae406d8e3ad1a/cmake/sip_configure.py#L191)?

Reply:
Sure that will be possible. But, I will not have cycles to do that in any new future. I am using the present patch since several months already and just barely managed to file the PR now.